rmgpy/molecule/group.py

```diff
@@ -1392,7 +1440,7 @@ def classifyBenzeneCarbons(self, partners = None):
         #check that cbfAtoms only have benzene bonds
         for cbfAtom in cbfAtomList:
             for atom2, bond12 in cbfAtom.bonds.iteritems():
-                assert bond12.isBenzene(), "Cbf atom in {0} has a bond with an order other than 'B'".format(self)
+                assert bond12.isBenzene(), "Cbf atom in {0} has a bond with an order other than 1.5".format(self)
```

Comment:
The bond orders for Cbf atoms has always been a bit troublesome, but it's highlighted now because the bond orders are more explicit. When we made the changes to benzene bond treatment somewhat recently, we considered benzene bonds around a Cbf atom to have bond orders of 1.33. Switching to numerical bond orders makes this issue a bit more difficult to reconcile.